### PR TITLE
Added Airsonic

### LINKED ISSUE #248: airsonic
Info about it:
https://github.com/airsonic/airsonic

Docker and integration:
https://github.com/linuxserver/docker-airsonic
https://github.com/linuxserver/docker-airsonic-armhf
https://github.com/linuxserver/docker-airsonic-arm64

Reason:
FOSS music server / very popular

### PR DESCRIPTION
**Pull request by hijxf**
## Purpose
Adds Airsonic to DS for alternative music streaming. Closes #248 

## Approach
Adds new container to project

#### Open Questions and Pre-Merge TODOs
- [x] Needs testing
- [x] Add to available apps wiki after merge

#### Learning
https://github.com/GhostWriters/DockSTARTer/pull/211/files
https://hub.docker.com/r/lsioarmhf/airsonic/

## Requirements
- [x] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: compose/.apps/airsonic/airsonic.yml [hunk 1]
```diff
@@ -1,19 +1,18 @@
 services:
   airsonic:
-    image: lsioarmhf/airsonic
-    container_name: airsonic
-    restart: always
+    image:           lsioarmhf/airsonic
```
Comment (hijxf) [inline compose/.apps/airsonic/airsonic.yml]: Thanks :)

File: compose/.apps/airsonic/airsonic.netmode.yml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 services:
-  heimdall:
-    network_mode: ${AIRSONIC_NETWORK_MODE}
+  airsonic:
```
Comment (hijxf) [inline compose/.apps/airsonic/airsonic.netmode.yml]: hahaha oops

**Comment by hijxf** [comment]
thank god for PRs

**Comment by nemchik** [comment]
Also FYI i changed the title and edited the Purpose section of the first PR message to include the closing remark. This way GitHub automation actually kicks in and closes the other ticket automatically (notice the word `Closes` up there is underlined and hovering displays the special message).

**Comment by nemchik** [comment]
Would you mind pulling these new changes to your local system and testing the branch to confirm the app works as it should?

**Comment by hijxf** [comment]
yup, i'll do that tonight and leave a comment.

**Comment by hijxf** [comment]
works with armhf!

**Comment by hijxf** [comment]
maybe i should add some nginx confs before pushing into master

**Comment by nemchik** [comment]
nginx confs go here https://github.com/linuxserver/docker-letsencrypt/

if everything is working for you I'd say we're good to merge.

**Pull request merged**